src/capture/middlewares/tap_middleware/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+include_directories (
+  "${PROJECT_SOURCE_DIR}/src"
+)
+
+add_library(tap_middleware tap_middleware.c)
+target_include_directories(tap_middleware PRIVATE ${PROJECT_BINARY_DIR})
+target_link_libraries(tap_middleware PUBLIC PCAP::pcap PRIVATE pcap_service log os SQLite::SQLite3)
```

```suggestion
target_link_libraries(tap_middleware PUBLIC SQLite::SQLite3 pcap_service eloop PRIVATE allocs os log squeue)
```

`tap_middleware.h` includes `middleware.h`, which includes `SQLite::SQLite3 pcap_service eloop`. Ideally, we'd probably want to make a `middleware` CMake target, but that can be a future refactor.